.github/workflows/package.yml

```diff
@@ -167,6 +187,17 @@ jobs:
         with:
           node-version: '22.x'
           registry-url: 'https://registry.npmjs.org'
+      - name: Publish Snap to Snap Store
```

Comment:
I propose we first publish the snaps to GitHub Releases. Once we make sure we got that right and that it all works well, we can look into publishing them to the official store. That way we can test all of this a few times before hitting the store for real